docs/wiki/[Variables]-configure_management_resources.md

```diff
@@ -1 +1,355 @@
+        enable_solution_for_vm_insights             = bool
+        enable_sentinel                             = bool
+      })
+    })
+    security_center = object({
+      enabled = bool
+      config = object({
+        email_security_contact             = string
+        enable_defender_for_acr            = bool
+        enable_defender_for_app_services   = bool
+        enable_defender_for_arm            = bool
+        enable_defender_for_dns            = bool
+        enable_defender_for_key_vault      = bool
+        enable_defender_for_kubernetes     = bool
+        enable_defender_for_oss_databases  = bool
+        enable_defender_for_servers        = bool
+        enable_defender_for_sql_servers    = bool
+        enable_defender_for_sql_server_vms = bool
+        enable_defender_for_storage        = bool
+      })
+    })
+  })
+  location = any
+  tags     = any
+  advanced = any
+})
+```
+
+## Usage
+
+Configure resources for the `management` Landing Zone, including:
+
+- Central Log Analytics workspace
+- Azure Sentinel
+- Microsoft Defender for Cloud (previously Security Center/Azure Defender)
+
+### Configure Log Analytics
+
+Define the creation of a central Log Analytics workspace and the associated monitoring solutions.
+
+```hcl
+log_analytics = {
+  enabled = true
+  config = {
+    retention_in_days                           = 30
+    enable_monitoring_for_arc                   = true
+    enable_monitoring_for_vm                    = true
+    enable_monitoring_for_vmss                  = true
+    enable_sentinel                             = true
+    enable_solution_for_agent_health_assessment = true
+    enable_solution_for_anti_malware            = true
+    enable_solution_for_azure_activity          = true
+    enable_solution_for_change_tracking         = true
+    enable_solution_for_service_map             = true
+    enable_solution_for_sql_assessment          = true
+    enable_solution_for_updates                 = true
+    enable_solution_for_vm_insights             = true
+  }
+}
+```
+
+This object allows you to configure the data retention period and the deployed Log Analytics solutions.
+
+#### `settings.log_analytics.enabled`
+
+The `enabled` (`bool`) input allows you to toggle whether to create the Log Analytics workspace.
+Note that this is required for many of the Azure Policy assignments to function properly.
+For example, the deploy diagnostic settings policies require a Log Analytics workspace as an assignment parameter.
+
+#### `settings.log_analytics.config`
+
+The `config` (`object`) input allows you to set the following configuration items for the Log Analytics workspace:
+
+##### `settings.log_analytics.config.retention_in_days`
+
+The number of days to retain data in the Log Analytics workspace.
+
+##### `settings.log_analytics.config.enable_monitoring_for_arc`
+
+Enables the following Azure Policy assignments at your organizational root management group scope:
```

Comment:
I think we're trying to use the term "intermediate root"

Reply:
👍